app/views/Project/EntryEdit/index.tsx

```diff
@@ -1589,26 +1590,42 @@ function EntryEdit(props: Props) {
-                                        spacing="comfortable"
-                                        rendererParams={entryDataRendererParams}
-                                        filtered={false}
-                                        errored={false}
-                                        pending={false}
-                                        emptyIcon={(
-                                            <Kraken
-                                                variant="search"
+                                    {(formValue?.entries?.length ?? 0) > 0
+                                        ? (
+                                            <VirtualizedListView
+                                                itemHeight={360}
+                                                keySelector={entryKeySelector}
+                                                renderer={EntryInput}
+                                                data={formValue.entries}
+                                                direction="vertical"
+                                                spacing="comfortable"
+                                                rendererParams={entryDataRendererParams}
+                                                filtered={false}
+                                                errored={false}
+                                                pending={false}
+                                                emptyIcon={(
+                                                    <Kraken
+                                                        variant="search"
+                                                    />
+                                                )}
+                                                emptyMessage="No entries found"
+                                                messageIconShown
+                                                messageShown
                                             />
+                                        ) : (
+                                            <div className={styles.noEntriesFound}>
+                                                <Message
+                                                    icon={(
+                                                        <Kraken
+                                                            variant="search"
+                                                            size="large"
+                                                        />
+                                                    )}
+                                                    message="No entries found."
+                                                />
+                                            </div>
                                         )}
-                                        emptyMessage="No entries found"
-                                        messageIconShown
-                                        messageShown
-                                    />
+
```

Comment:
VirtualizedListView should handle emptyMessage

Reply:
The mentioned issue required that the empty message should appear at the center of the component and to fix this issue a separate component hiding the layer of <VirtualizedListView /> was used.